### PR TITLE
cino-= and misc style changes: indent.vim

### PR DESCRIPTION
**Pull request by bounceme**

### CODE REVIEW

File: indent/javascript.vim [hunk 1]
```diff
@@ -458,7 +459,7 @@ function GetJavascriptIndent()
   if l:line =~ '^[])}]\|^|}'
     return max([indent(num),0])
   elseif num
-    return indent(num) + s:W + switch_offset + bL + isOp
+    return max([indent(num) + get(l:,'case_offset',s:W) + switch_offset + bL + isOp, 0])
```
Comment (utix) [inline indent/javascript.vim]: I think case_offset visibility should be change as s`witch_offset`. Just set it line 431 to` s:sw()`.
Or better way, set it to `0` line 431 and line 440 to  `cino(`=`) -s:W` . So line 462  should be
`return max([indent(num) + s:W + switch_offset + +case_offset + bL + isOp, 0])`

Comment (bounceme) [inline indent/javascript.vim]: case_offset, if existing, replaces the &sw which is added for lines between `{}`. it isn't additional indent

Comment (bounceme) [inline indent/javascript.vim]: the switch_offset is also added

Comment (bounceme) [inline indent/javascript.vim]: i like to have the additional variable, `s:W` is used in other functions between the main return and the switch part

**Pull request merged**